### PR TITLE
Server: minimum setup for development

### PR DESCRIPTION
**Pull request by fjebaker**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,17 @@
...
+
+    pip install --user pdm
+
+Then install the dependencies
+
+    pdm install
+
+Finally start the server
+
+    pdm run serve
```
Comment (JR-Mitchell) [inline README.md:13]: Is this a typo?

Comment (fjebaker) [reply README.md:13]: NEIN

Comment (Shellywell123) [reply README.md:13]: I like swerve
thoughts?

**Comment by JR-Mitchell** [comment]
LGTM

**Review by JR-Mitchell — APPROVED** [review]

**Pull request merged**